src/main/kotlin/app/revanced/patches/shared/integrations/AbstractIntegrationsPatch.kt

```diff
@@ -1,20 +1,18 @@
-package app.revanced.patches.shared.integrations.patch
+package app.revanced.patches.shared.integrations
 
-import app.revanced.patcher.annotation.Description
 import app.revanced.patcher.data.BytecodeContext
 import app.revanced.patcher.extensions.InstructionExtensions.addInstruction
 import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
 import app.revanced.patcher.patch.BytecodePatch
 import app.revanced.patcher.patch.PatchException
-import app.revanced.patches.shared.integrations.patch.AbstractIntegrationsPatch.IntegrationsFingerprint.RegisterResolver
+import app.revanced.patches.shared.integrations.AbstractIntegrationsPatch.IntegrationsFingerprint.RegisterResolver
 import com.android.tools.smali.dexlib2.Opcode
 import com.android.tools.smali.dexlib2.iface.ClassDef
 import com.android.tools.smali.dexlib2.iface.Method
 
-@Description("Applies mandatory patches to implement the ReVanced integrations into the application.")
```

Comment:
This is not a patch class and instead abstract, and as it is not singleton the annotation processor would fail to compile the generated class. Patches that are annotated with @Patch neither need a name nor description, as those are metadata that are useful for displaying patches only.